corehq/util/model_log.py

```diff
@@ -31,6 +40,6 @@ def log_model_change(user, model_object, message=None, fields_changed=None, is_c
         content_type_id=get_content_type_for_model(model_object).pk,
         object_id=model_object.pk,
         object_repr=force_text(model_object),
-        action_flag=ADDITION if is_create else CHANGE,
+        action_flag=action,
```

Comment:
```suggestion
        action_flag=action.value,
```

Reply:
Ah! thanks. Forgot to change that, was just planning on using direct constants earlier.